include/aws/http/http2_stream_manager.h

```diff
@@ -51,9 +51,12 @@ struct aws_http2_stream_manager_options {
     /**
      * If TLS options is set, you also need to handle ALPN, otherwise, may not able to get HTTP/2 connection and fail
      * the stream manager.
-     * If TLS options not set, prior knowledge will be used.
+     * If TLS options not set, you can set prior_knowledge to true to establish HTTP/2 connection. We don't support
+     * upgrade from HTTP/1 connection.
```

Comment:
trivial: ~We don't support~
maybe: Upgrade from HTTP/1 connection is not supported.

It's fine to say "you" when talking to users, but don't use "we" in the public documentation (it's fine to use in C files)